### ISSUE TITLE
deprecate (add warnings) to any scss code that has been duplicated into deprecated-component-library-helpers

### ISSUE THREAD

**Issue by mbylstra** [cultureamp/kaizen-design-system#1010]
There is currently a lot of duplicated code in https://github.com/cultureamp/kaizen-design-system/tree/master/packages/deprecated-component-library-helpers/styles and https://github.com/cultureamp/kaizen-design-system/tree/master/packages/component-library/styles.

This makes it very hard for contributers to know which ones to update (one, the other, or both?). Let's try to add deprecation warnings as suggested in https://github.com/cultureamp/kaizen-design-system/issues/701 to make this problem more noticeable.

TODO:
- [ ] work out exactly which variables and mixins in the component-library are deprecated. The ones that are duplicated in deprecated-component-library-helpers are 100% deprecated, but the ones that are it's less clear. Pretty much all of the  component-library/styles is deprecated in the sense that it's not the way we want to do things going forward, but they the maybe still useful and widely used without a better alternative yet (so not really deprecated yet)
- [ ] figure out an effective way (as suggested in https://github.com/cultureamp/kaizen-design-system/issues/701)
- [ ] add comments to the top of both files about whether the corresponding duplicate code needs to be updated  (yes it does)

Stretch goal (split into new card issue if not done):

- [ ] see if there are any differences between sass code in component-library/styles and deprecated-component-library-helpers (using diff program). If there are, then "the conflicts" need to be "resolved" (this is the big problem with having duplicated code!) as we can't easily be sure which repos are using the older code (which might have drifted out of sync.

Once this is done, we want to also tackle https://github.com/cultureamp/kaizen-design-system/issues/1012 (this is just a stepping stone)

**Comment by mbylstra** [comment]
Done in https://github.com/cultureamp/kaizen-design-system/pull/1046
